.circleci/config.yml

```diff
@@ -165,6 +165,8 @@ jobs:
       - persist_to_workspace:
           root: .
           paths:
+            - tmp/cache/webpacker/last-compilation-digest-test
+            - tmp/cache/assets
```

Comment:
curiouly, these lines make it pass (though the CI run itself does not end up being meaningfully faster)

Reply:
It's confusing because `sassc` shouldn't even really be attempting to run while CI is running specs since we precompile and then disable webpacker compile, but it must do some kind of check against the cache that is still able to segfault if this resolves it.